Task5/index.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Voyage</title>
+        <meta charset="utf-8">
+        <link rel="stylesheet" type="text/css" href="css/header/__background-image.css">
+        <link rel="stylesheet" type="text/css" href="css/header/menu/__item.css">
+        
+        <link rel="stylesheet" type="text/css" href="css/content.css">
+        <link ref="stylesheet" type="text/css" href="css/header/header.css">
+        <link rel="stylesheet" type="text/css" href="css/footer/footer.css">
+        <link rel="stylesheet" type="text/css" href="css/footer/table/__col.css">
+        <link rel="stylesheet" type="text/css" href="css/footer/table/__info.css">
+        <link rel="stylesheet" type="text/css" href="css/__.css">        
+        <link rel="stylesheet" type="text/css" href="css/footer/table/col2/__contact.css">
+        <link rel="stylesheet" type="text/css" href="css/footer/table/col2/__icon.css">
+        <link rel="stylesheet" type="text/css" href="css/templates/font.css">
+        <link rel="stylesheet" type="text/css" href="css/templates/button.css">
+        <link rel="stylesheet" type="text/css" href="css/templates/selected-item.css">
+        <link rel="stylesheet" type="text/css" href="css/header/__content.css">
+        <link rel="stylesheet" type="text/css" href="css/header/label/__logo.css">
+        <link rel="stylesheet" type="text/css" href="css/header/__label.css">
+        <link rel="stylesheet" type="text/css" href="css/header/label/__headline.css">
+        <link rel="stylesheet" type="text/css" href="css/header/label/__button.css">
+        <link rel="stylesheet" type="text/css" href="css/main/__grid.css">
+        <link rel="stylesheet" type="text/css" href="css/templates/bestseller-label.css">
+        <link rel="stylesheet" type="text/css" href="css/templates/sale-label.css">
+        <link rel="stylesheet" type="text/css" href="css/main/__button.css">
+        <link rel="stylesheet" type="text/css" href="css/main/__catalog-text.css">
+        <link rel="stylesheet" type="text/css" href="css/main/__sort-text.css">
+        <link rel="stylesheet" type="text/css" href="css/main/__show-more-img.css">
+        <link rel="stylesheet" type="text/css" href="css/main/tour/__image.css">
+        <link rel="stylesheet" type="text/css" href="css/main/main.css">
+        <link rel="stylesheet" type="text/css" href="css/main/tour/__name.css">
+        <link rel="stylesheet" type="text/css" href="css/main/tour/__info.css">
+        <link rel="stylesheet" type="text/css" href="css/main/tour/__old-price.css">
+        <link rel="stylesheet" type="text/css" href="css/main/tour/__price.css">
+    </head>
+    <body>
+        <header >
+            <div class="content header__content">
+                <div class="header__menu-logo">
+                    <img src="images/меню.png">
```

Comment:
As I mentioned earlier, Russian characters shouldn't be used in urls.

Reply:
Renamed